README.md

```diff
@@ -162,21 +164,21 @@ on the **decoded** token before allowing the visitor to proceed.
 
 - `key` - (***required*** - *unless you have a `customVerify` function*) the secret key (or array of potential keys)
 used to check the signature of the token ***or*** a **key lookup function** with
-signature `function(decoded, callback)` where:
+signature `async function(decoded)` where:
     - `decoded` - the ***decoded*** but ***unverified*** JWT received from client
-    - `callback` - callback function with the signature `function(err, key, extraInfo)` where:
-        - `err` - an internal error
+    - Returns an object `{ valid, key, extraInfo }` where:
         - `key` - the secret key (or array of keys to try)
         - `extraInfo` - (***optional***) any additional information that you would like to use in
         `validateFunc` which can be accessed via `request.plugins['hapi-auth-jwt2'].extraInfo`
-- `validateFunc` - (***required***) the function which is run once the Token has been decoded with
- signature `function(decoded, request, callback)` where:
+- `validate` - (***required***) the function which is run once the Token has been decoded with
```

Comment:
The documentation changed to accept `validate` but it still uses `validateFunc` everywhere. We should migrate to use validate like in `hapi-auth-basic`